### ISSUE TITLE
Wipe entire logical drives

### ISSUE THREAD

**Issue by encrypt0r** [DevelopersTree/permadelete#34]
Having a way to conveniently wipe a logical drive can be very helpful for users especially when trying to wipe a thumb drive or other external storage devices.

**Comment by rasyidf** [comment]
And maybe with this step :

0. Initialize
1. Load Drive
2. Check Drive Type, make sure it's not System drive.
3. Calculate Size, make sure User concern about the data.
4. Confirm Wipe out {0} In Size.
5. Wiping Out Asynchronously
6. Done, give message to User

**Comment by encrypt0r** [comment]
@rasyidf thank you for your suggestion, would you like to implement it? I'll help you along the way 😁

**Comment by rasyidf** [comment]
@encrypt0r  No Problem, working on it.

**Comment by rasyidf** [comment]
Ah, It seems the program structure is too complicated to just add a feature.

maybe we could discuss here:

What approach will be used as this operation:

- [ ] Users Drag and Drop Logical.
- [ ] Add New Wizard for Listing Drives.

**Comment by encrypt0r** [comment]
@rasyidf I agree it takes a while to get familiarized with the code base, so why don't we start with something that might not need to change the UI very much. Create a `ShredDriveAsync` method in [`ShredderService`](https://github.com/DevelopersTree/permadelete/blob/31b0a6e980bdc853924544ef6c377484d8ce174c/Permadelete.FrontEnd/Services/ShredderService.cs). The algorithm could look like this:
1. Make sure the drive is not on an SSD (Use [`ShredderService.IsDriveSSD`](https://github.com/DevelopersTree/permadelete/blob/31b0a6e980bdc853924544ef6c377484d8ce174c/Permadelete.FrontEnd/Services/ShredderService.cs#L254) for that). [Here is why](https://github.com/DevelopersTree/permadelete/wiki/How-Permadelete-works#the-case-of-ssds).
1. Fill up the free space on the drive by generating junk files. You can take a look at [`ShredderService.OverWriteFile`](https://github.com/DevelopersTree/permadelete/blob/31b0a6e980bdc853924544ef6c377484d8ce174c/Permadelete.FrontEnd/Services/ShredderService.cs#L199) for more information about that. You can either reuse it, or write a new method that creates a new file with arbitary length that that contains random bytes and has a random name.
1. Shred all of the folders on the drive by using [`ShredderService.ShredFolderAsync`](https://github.com/DevelopersTree/permadelete/blob/31b0a6e980bdc853924544ef6c377484d8ce174c/Permadelete.FrontEnd/Services/ShredderService.cs#L119)
1. Shred all of the files in the root of the drive (the files in the folders are already taken care of by `ShredFolderAsync`) by using [`ShredderService.ShredFileAsync`](https://github.com/DevelopersTree/permadelete/blob/31b0a6e980bdc853924544ef6c377484d8ce174c/Permadelete.FrontEnd/Services/ShredderService.cs#L83).

After this step is done, then we can integrate it into the UI, in the meantime, you can create a temp button for yourself while you're testing the new feature.
